### PR TITLE
Correct Css selector paypal payment method.

### PR DESCRIPTION
**Pull request by mrkhoa99**
### Description (*)
1. Log into Admin
2. Go to Payment method config
3. Select PayPal Express Checkout

**Expected Result:**

**Actual Result:**

The Get Credentials from PayPal and Sandbox Credentials button are broken.

Magento Admin 2019-07-19 15-34-13

### Fixed Issues (if relevant)

1. magento/magento2#<issue_number>: Issue title

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by mrkhoa99** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @mrkhoa99. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @mrkhoa99, here is your [Magento instance](https://i-23772-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-23772-2-3-develop.instances.magento-community.engineering/admin](https://i-23772-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by mrkhoa99** [comment]
I could see the issue with 2.3-develop instance

**Comment by mrkhoa99** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @mrkhoa99. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @mrkhoa99, here is your [new Magento instance](https://pr-23772.instances.magento-community.engineering).
Admin access: [https://pr-23772.instances.magento-community.engineering/admin](https://pr-23772.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by mrkhoa99** [comment]
@VladimirZaets could you please take a look at my PR? Thank you.

**Comment by sivaschenko** [comment]
@magento run all tests

File: app/design/adminhtml/Magento/backend/web/css/styles-old.less [hunk 1]
```diff
@@ -3962,7 +3962,7 @@
 }
 
 //  Configuration -> Payment Methods
-.adminhtml-system-config-edit {
+.adminhtml-system_config-edit {
```
Comment (sivaschenko) [inline app/design/adminhtml/Magento/backend/web/css/styles-old.less:3966]: Let's keep existing selector for backward compatibility
```suggestion
.adminhtml-system-config-edit,
.adminhtml-system_config-edit {
```

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Hi @mrkhoa99 thanks for the pull request, please take a look at my suggestion

File: app/design/adminhtml/Magento/backend/web/css/styles-old.less (hunk 1 above)
Comment (mrkhoa99) [reply app/design/adminhtml/Magento/backend/web/css/styles-old.less:3966]: thanks, changed it.

**Review by sivaschenko — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko, thank you for the review.
[ENGCOM-5490](https://jira.corp.magento.com/browse/ENGCOM-5490) has been created to process this Pull Request
:eight_spoked_asterisk: @sivaschenko, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

**Pull request merged**